### PR TITLE
Clarify release process

### PR DESCRIPTION
**Pull request by davejrt**
We discussed this in our meeting, and I advocated for delaying the release, however in hindsight I think as we have always done, it's better to have the 20th as the last possible day a release will be done.

### CODE REVIEW

**Review by daxmc99 — COMMENTED** [review]
Looks good, I think we should specify timezones though since "day" isn't the clearest language

File: handbook/engineering/releases/index.md [hunk 1]
```diff
@@ -128,6 +128,9 @@ This is because:
 
 In the future, we may introduce continuous releases if these issues become surmountable.
 
-### Weekend releases
 
-If the 20th falls on a non-working day, the release captain will move the release forward to the first working day. The calendar events will reflect this.
+### Important dates
+
+The release branch will be cut on the last working day before the 20th of each month. If the 20th falls on a non-working day, the release captain will move the release forward to the first working day, as well as cutting the release branch the day before. The calendar events will reflect this.
```
Comment (daxmc99) [inline handbook/engineering/releases/index.md:134]:
```suggestion
The release branch will be cut on the last working day at 10am PST before the 20th of each month. If the 20th falls on a non-working day, the release captain will move the release forward to the first working day, as well as cutting the release branch the day before. The calendar events will reflect this.
```

File: handbook/engineering/releases/index.md [hunk 2]
```diff
@@ -128,6 +128,9 @@ This is because:
...
 In the future, we may introduce continuous releases if these issues become surmountable.
 
-### Weekend releases
 
-If the 20th falls on a non-working day, the release captain will move the release forward to the first working day. The calendar events will reflect this.
+### Important dates
+
+The release branch will be cut on the last working day at 10am PST before the 20th of each month. If the 20th falls on a non-working day, the release captain will move the release forward to the first working day, as well as cutting the release branch the day before. The calendar events will reflect this.
+
+eg. The 20th of a given month falls on a Sunday, therefore the release will be moved forward to the first working day (the 18th) and the branch will cut the day before (the 17th).
```
Comment (bobheadxi) [inline handbook/engineering/releases/index.md:136]: q (was absent for the meeting): are we moving away from the 1-day release we were aiming for before? (https://github.com/sourcegraph/sourcegraph/pull/22773#issuecomment-878801279)

Comment (davejrt) [reply handbook/engineering/releases/index.md:136]: For time time being, yes. Until such time as CI doesn't take 25-30 minutes to run each time, we agreed it's better to spread this out. I've also encountered issues in the past where CI has broken on the release branch and it causes undue stress on the release captain as distribution don't own any of the tests or code, and it isn't always easy to get people in a different timezone to fix things.

Comment (bobheadxi) [reply handbook/engineering/releases/index.md:136]: thank you! (just watched the meeting and heard the discussion, agreed)

**Pull request merged**